src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp

```diff
@@ -66,5 +67,15 @@ struct ExcisionSphere : db::SimpleTag {
   }
 };
 
+/*!
+ * \brief A map that holds the grid coordinates of element faces abutting the
+ * worldtube with the corresponding ElementIds.
+ */
+template <size_t Dim>
+struct ElementFacesCoordinatesMap : db::SimpleTag {
```

Comment:
Maybe name this `ElementFacesGridCoordinates`? `CoordinatesMap` to me is confusing with our actually coordinate map objects.